### DISCUSSION TITLE
Yarn dev failed for nextjs 13 when create next-app

### DISCUSSION THREAD

**Discussion by huynguyen2k** [vercel/next.js#43911] [Help]
Today when I create nextjs app version 13 with yarn then I have this problem. Is anyone have this problem like me and how to resolve it? Thanks!

![Capture](https://user-images.githubusercontent.com/80769479/206828285-1246057e-bf4c-4cf6-a526-9ca9525581b1.PNG)
![Capture1](https://user-images.githubusercontent.com/80769479/206828329-b9e1d580-cefc-404d-a978-5150c9a2c40e.PNG)

**Comment by ciruz** [comment]
I use ``yarn dev`` everyday, no problem. Do you have an example repo with this error?
